packages/terra-clinical-data-grid/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Added prop `overflowContainerRefCallback`.
```

Comment:
I'd be a bit more descriptive — as in, what does it do?

Reply:
Fixed here https://github.com/cerner/terra-clinical/pull/585/commits/f40141f133c81d20aaa0b6004cb721b5c311ac71.